src/store/printer/getters.ts

```diff
@@ -760,7 +761,6 @@ export const getters: GetterTree<PrinterState, RootState> = {
       'aht10',
       'bme280',
       'htu21d',
-      'nevermoresensor',
```

Comment:
This will break VOC plotting on the temp graph.

Reply:
This is just extra sensor data (does not include temperature, just pressure, humidity and gas)